README.md

```diff
@@ -47,6 +66,14 @@ Future features include managing/creating email campaigns and ticket exchanges.
 2. Open PGAdmin and create a new server.
 3. Set the credentials to the values in the `.env` file.
 
+### Using Swagger:
+1. To get the bearer token, create a user by going through the signup process. 
+   - For admin functions, make sure the user has an admin role (contact team lead for admin role).
```

Comment:
Just a curious question but how are admin roles created/given?

Reply:
Inside of Auth0, under users, an auth0 admin can elevate privileges for individual users